tests/examples/source/nodejs/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/phmartin/node:17
+FROM quay.io/odo-dev/node:19
```

Comment:
@anandrkskd @HarshithaMS005 It is not clear to me how this base image (`quay.io/odo-dev/node:19`) was built. Could we add a comment (or better, create another Dockerfile, say `base.dockerfile`) with instructions on how to build this base image? I think it would be helpful in the future if we need to update this base image.

Reply:
@rm3l we just copied the image from docker hub to quay.io, we didn't build the image from some dockerfile, if you want we can add the command as a comment for refference

Reply:
@HarshithaMS005 can you add the command as comment

Reply:
Yes i have added